src/AspNetCoreModuleV2/InProcessRequestHandler/managedexports.cxx

```diff
@@ -94,7 +94,8 @@ http_set_response_status_code(
     _In_ PCSTR pszReason
 )
 {
-    return pInProcessHandler->QueryHttpContext()->GetResponse()->SetStatus(statusCode, pszReason);
+    return pInProcessHandler->QueryHttpContext()->GetResponse()->SetStatus(statusCode, pszReason, 0, 0, nullptr,
```

Comment:
I don't see this overload here: https://msdn.microsoft.com/en-us/library/ms690943(v=vs.90).aspx. Undocumented?

Reply:
Also, do we care about returning the hresult?

Reply:
Seems to be undocumented, why wouldn't we care?

Reply:
I don't think we care, it just sucks that it isn't discoverable.

Reply:
Why isn't it? We check it on managed side. We can also wrap them into LOG_IF tho.

Reply:
No log info as it's on a request basis.

Reply:
Ping sourabh, we should get any APIs we use documented.

Reply:
This is known issues. The API is in the public header file. The documentation for couple iis APIs need to be updated.